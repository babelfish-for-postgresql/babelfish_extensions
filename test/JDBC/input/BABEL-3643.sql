sp_helpdb 'MaSTeR'
go

sp_helpdb 'master'
go
