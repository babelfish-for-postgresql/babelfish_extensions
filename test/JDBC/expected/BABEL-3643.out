sp_helpdb 'MaSTeR'
go
~~START~~
nvarchar#!#nvarchar#!#nvarchar#!#int#!#nvarchar#!#nvarchar#!#tinyint
master#!#<NULL>#!#jdbc_user#!#1#!#2023-03-08 #!#<NULL>#!#<NULL>
~~END~~

~~START~~
nchar#!#smallint#!#nchar#!#nvarchar#!#nvarchar#!#nvarchar#!#nvarchar#!#varchar
<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>
~~END~~


sp_helpdb 'master'
go
~~START~~
nvarchar#!#nvarchar#!#nvarchar#!#int#!#nvarchar#!#nvarchar#!#tinyint
master#!#<NULL>#!#jdbc_user#!#1#!#2023-03-08 #!#<NULL>#!#<NULL>
~~END~~

~~START~~
nchar#!#smallint#!#nchar#!#nvarchar#!#nvarchar#!#nvarchar#!#nvarchar#!#varchar
<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>
~~END~~

